android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,7 @@
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.4-bin.zip
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.5-bin.zip
+networkTimeout=10000
+validateDistributionUrl=true
```

Comment:
Can we migrate these gradle changes to a separate PR unless the changes are needed for this fix?

Reply:
App does not build without upgrading

Reply:
Yeah lets remove the gradlew stuff from this PR. CI should build fine.

Reply:
Reverted